### DISCUSSION TITLE
启动光源，vc-graphics-polygon材质颜色会更改。

### DISCUSSION THREAD

**Discussion by Fang-yl** [zouyaoji/vue-cesium#636] [Q&A]
启动光源，使用vc-graphics-polygon，设置material为#fff，shadows为1接受并投射光源，超光面材质颜色会更改，按道理朝光面颜色应该不会更改，背光面会更改。设置shadows为 2只投射，材质颜色正常。

**Comment by zouyaoji** [comment]
没有做特别的处理哈，只是转换成组件了，这应该是cesium的PolygonGraphics就是这样的表现。

**Comment by Fang-yl** [comment]
好的，那请问一下光照阴影的颜色如何更改，比如早上六七点钟的颜色很淡，跟影像颜色很相近，无法看出是否有阴影，请问是否有相关的api设置。

**Reply by zouyaoji** [reply]
viewer上有api可以关闭阴影的。

**Reply by Fang-yl** [reply]
大佬，不是想要关闭阴影的，就是想让阴影颜色固定，不会随时间变化颜色变淡，请问一下现在有啥api配置么。

**Reply by zouyaoji** [reply]
没搞过，可以试试这个参数
![image](https://github.com/user-attachments/assets/207fbffc-09a3-4edc-87e9-cb3f8c45672d)

**Reply by Fang-yl** [reply]
好的，感谢大佬，我来试试。
